test/e2e/manifests/tke-platform-controller/tke-platform-controller.yaml

```diff
@@ -28,13 +32,15 @@ spec:
           mountPath: /app/conf
         - name: provider-volume
           mountPath: /app/provider/baremetal
+        - name: containerd-volume
+          mountPath: /app/provider/baremetal/conf/containerd/
         - name: provider-config-volume
           mountPath: /app/provider/baremetal/conf/
         - name: docker-volume
           mountPath: /app/provider/baremetal/conf/docker/
         - name: kubelet-volume
           mountPath: /app/provider/baremetal/conf/kubelet/
-        - name: kubeadm-volume
+        - name: kubeadm-config-volume
```

Comment:
So we cannot upgrade just replace images?

Reply:
TODO on next PR for upgrade case.